components/eventing-controller/pkg/deployment/publisher_deployment.go

```diff
@@ -316,6 +316,10 @@ func getBEBEnvVars(publisherConfig env.PublisherConfig) []v1.EnvVar {
 			Name:  "BEB_NAMESPACE",
 			Value: fmt.Sprintf("%s$(BEB_NAMESPACE_VALUE)", bebNamespacePrefix),
 		},
+		{
+			Name:  "ENABLE_NEW_CRD_VERSION",
+			Value: strconv.FormatBool(publisherConfig.EnableNewCRDVersion),
+		},
```

Comment:
why do you need this?

Reply:
I thought event if Event Mesh is enabled, the `/subscribed`endpoint should be callable.

Reply:
how does the `/subscribed` endpoint depend on the backend enabled?